components/locid/src/extensions/mod.rs

```diff
@@ -176,6 +213,9 @@ impl writeable::Writeable for Extensions {
         result += writeable::Writeable::write_len(&self.transform);
         result += writeable::Writeable::write_len(&self.unicode);
         result += writeable::Writeable::write_len(&self.private);
+        self.other
+            .iter()
+            .for_each(|other| result += writeable::Writeable::write_len(other));
```

Comment:
Suggestion (optional):

```suggestion
        result += self.other.iter().map(writeable::Writeable::write_len).sum();
```